map/framework.cpp

```diff
@@ -2501,31 +2515,97 @@ bool Framework::GetEditableMapObject(FeatureID const & fid, osm::EditableMapObje
   osm::Editor & editor = osm::Editor::Instance();
   emo.SetEditableProperties(editor.GetEditableProperties(ft));
 
+  // Get exact feature's street address (if any) from mwm,
+  // together with all nearby streets.
+  search::ReverseGeocoder const coder(m_model.GetIndex());
+  auto const streets = coder.GetNearbyFeatureStreets(ft);
+
+  // The code below resolves conflicts between editro's
+  // and mwm's version of a feature. emo should contain
+  // localized streets that are nearby to a given feature
+  // and the street a given feature is assigned to.
+  // Such a street may or may not be set by a user, may
+  // or may not be set in mwm and can be localized only
+  // if it is localized in mwm. If a street is set by a user
+  // it can be localized if it equals to one of nearby
+  // to a given feature streets.
+
   string street;
-  if (editor.GetEditedFeatureStreet(fid, street))
+  auto const featureIsInEditor = editor.GetEditedFeatureStreet(fid, street);
+  auto const featureHasStreetInMwm = streets.second < streets.first.size();
```

Comment:
Почему не bool?

> On 14.4.2016, at 16.25, mgsergio <notifications@github.com> wrote:
> 
> In map/framework.cpp:
> 
> >    string street;
> > -  if (editor.GetEditedFeatureStreet(fid, street))
> > +  auto const featureIsInEditor = editor.GetEditedFeatureStreet(fid, street);
> > +  auto const featureHasStreetInMwm = streets.second < streets.first.size();
> auto?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>